### PR TITLE
Added d2l summary view with demos

### PR DESCRIPTION
**Pull request by m6jones**
image

### CODE REVIEW

File: components/d2l-enrollment-summary-view/d2l-enrollment-summary-view-layout.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+					.desvl-flex {
+						margin: 0;
+						padding: 0 0.9rem;
+					}
+				}
+			</style>
+			<div class="desvl-center">
+				<div class="desvl-flex">
+					<span class="desvl-first-column">
+						<slot name="first-coloumn"></slot>
```
Comment (AdmiralCoco) [inline components/d2l-enrollment-summary-view/d2l-enrollment-summary-view-layout.js]: Typo: `coloumn` -> `column`, also line 67

File: components/d2l-enrollment-summary-view/d2l-enrollment-summary-view-tag-list.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ */
+class D2lEnrollmentSummaryViewTagList extends PolymerElement {
+	static get template() {
+		return html`
+			<style>
+				:host {
+					@apply --d2l-body-small-text;
+					color: var(--d2l-color-tungsten);
+				}
+				.desvtl-contianer {
```
Comment (AdmiralCoco) [inline components/d2l-enrollment-summary-view/d2l-enrollment-summary-view-tag-list.js]: Typo: `contianer` -> `container`, also see other classes in this file

File: components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+					}
+				}
+			</style>
+			<div class="desv-header">
+				<div class="desv-title-bar">
+					<h1> Introduction to User Experience Design </h1>
+					<d2l-enrollment-summary-view-tag-list list=[[_tags]]></d2l-enrollment-summary-view-tag-list>
+				</div>
+				<d2l-enrollment-summary-view-layout>
+					<div slot="first-coloumn">Completion Bar</div>
```
Comment (AdmiralCoco) [inline components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js]: Typo: `column`*

File: components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+		return [
+			'_handleEnrollmentResponse(entity)'
+		];
+	}
+	_computeTags(courses) {
+		const tags = [];
+		if (courses) {
+			tags.push(courses.length > 1 ? courses.length + ' Activities' : '1 Activity');
+		}
+		tags.push('About 2 hour 30 minutes');
```
Comment (AdmiralCoco) [inline components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js]: Are these strings meant to be constant/localized?

File: components/d2l-enrollment-summary-view/d2l-enrollment-summary-view-layout.js (hunk 1 above)
Comment (m6jones) [reply components/d2l-enrollment-summary-view/d2l-enrollment-summary-view-layout.js]: Thank you :)

File: components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js (hunk 4 above)
Comment (m6jones) [reply components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js]: Yeah for now constant/mocked

**Pull request merged**